worker/raft/rafttest/fixtures.go

```diff
@@ -74,6 +74,7 @@ func (s *RaftFixture) NewRaft(c *gc.C, id raft.ServerID, fsm raft.FSM) (
 
 func (s *RaftFixture) DefaultConfig(id raft.ServerID) *raft.Config {
 	raftConfig := raft.DefaultConfig()
```

Comment:
If this is turned off in the real workers, should this be set in DefaultConfig()?

Reply:
Oh, that's a library function.